privaterelay/templates/includes/header.html

```diff
@@ -15,6 +15,9 @@
     </div>
   {% elif not request.user.is_anonymous %}
     <div id="micro-survey-banner" class="micro-survey-banner hidden">
+      <button id="survey-dismiss" class="flx center-value dismiss">
```

Comment:
I want to retire the utility classes going forward for all new code. Can you move the `flx center-value` into the SCSS for this class instead?